crates/typst/src/model/label.rs

```diff
@@ -29,8 +32,8 @@ use crate::eval::{func, scope, ty, Repr};
 /// Currently, labels can only be attached to elements in markup mode, not in
 /// code mode. This might change in the future.
 #[ty(scope)]
-#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
-pub struct Label(pub EcoString);
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Ord, PartialOrd, Hash)]
+pub struct Label(pub PicoStr);
```

Comment:
I think this should be a private field. Label can have 
- `fn new(string: impl Into<PicoStr>) -> Self`
- `as_str(self) -> &'static str`
- `into_inner(self) -> PicoStr`